`make test-playground` fails

Here is the result of the Jest testing:

```
 FAIL  __tests__/Editor.test.tsx
  ● Test suite failed to run

    src/components/WasmTest/Wasm.tsx:7:44 - error TS1343: The 'import.meta' meta-property is only allowed when the '--module' option is 'es2020', 'esnext', or 'system'.

    7     new Worker(new URL('./wasm.worker.ts', import.meta.url), {
                                                 ~~~~~~~~~~~

 FAIL  __tests__/Welcome.test.tsx
  ● Test suite failed to run

    src/components/WasmTest/Wasm.tsx:7:44 - error TS1343: The 'import.meta' meta-property is only allowed when the '--module' option is 'es2020', 'esnext', or 'system'.

    7     new Worker(new URL('./wasm.worker.ts', import.meta.url), {
                                                 ~~~~~~~~~~~

Test Suites: 2 failed, 2 total
Tests:       0 total
Snapshots:   0 total
Time:        63.12 s, estimated 90 s
```

--- a/packages/playground/webpack.config.js
+++ b/packages/playground/webpack.config.js
@@ -83,7 +83,9 @@ module.exports = {
       // available options are documented at https://github.com/Microsoft/monaco-editor-webpack-plugin#options
       languages: ['rust'],
     }),
-    new EnvironmentPlugin(['COMPILE_URL']),
+    new EnvironmentPlugin({
+      COMPILE_URL: '',
+    }),
   ],
   experiments: {
     asyncWebAssembly: true,

